simpleSequential/cube.test.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+from cube import cube
+
+class TestCubeFunction(unittest.TestCase):
+    def test_cubeFuctionCase1(self):
+        self.assertEqual(cube(2), 8, "Error: cube result is incorrect.")
+
+    def test_cubeFuctionCase2(self):
+        self.assertEqual(cube(-3), -27, "Error: cube result is incorrect.")
+
+    def test_cubeFuctionCase3(self):
+        self.assertEqual(cube(1), 1, "Error: cube result is incorrect.")
```

Comment:
Not bad, but I think 0 should be a better test.

Reply:
This test is sufficient because number 1 can prove that the cube is working better.